gcn/notices/icecube/LVK_nu_track_search.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "IceCube LVK Alert Nu Track Search",
+  "description": "IceCube LVK Coincident Neutrino Track Search",
+  "anyOf": [
+    {
+      "$ref": "/schema/gcn/notices/core/Alert.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/Event.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/DateTime.schema.json"
+    }
+  ],
+  "properties": {
+    "pval_generic": {
+      "description": "P-value from generic transient search; consistency with background expectations [0-1]",
+      "type": "number"
+    },
+    "pval_bayesian": {
+      "description": "P-value from LLAMA bayesian search; consistency with background expectations [0-1]",
+      "type": "number"
+    },
+    "n_events_coincident": {
+      "description": "Number of IceCube events in spatial and temporal coincidence with the GW map",
+      "type": "number"
+    },
+    "coincident_events": {
+      "type": "array",
+      "description": "An array of summary of information for each coincident neutrino track event",
+      "items": { "$ref": "#/$defs/coincevent" }
+    },
+    "most_probable_direction": {
+      "description": "Most likely source direction based on all coincident events and GW localizations",
+      "$ref": "/schema/gcn/notices/core/Localization.schema.json"
+    },
+    "neutrino_flux_sensitivity_range": {
+      "description": "Time integrated flux sensitivity ranges assuming an E^-2 spectrum (E^2 dN/dE) within the GW map localization",
+      "$ref": "#/$defs/fluxlimits"
+    }
+  },
+  "$defs": {
+    "fluxlimits": {
+      "type": "object",
+      "properties": {
+        "flux_sensitivity_max": {
+          "description": "Largest time integrated flux sensitivity [GeV cm^-2] assuming an E^-2 spectrum (E^2 dN/dE) found within the 90% region of GW map localization",
+          "type": "number"
+        },
+        "flux_sensitivity_min": {
+          "description": "Smallest time integrated flux sensitivity [GeV cm^-2] assuming an E^-2 spectrum (E^2 dN/dE) found within the 90% region of GW map localization",
+          "type": "number"
+        },
+        "sensitive_energy_range_lower": {
```

Comment:
"sensitive_energy_range": { 
"type":"array",
 "Lower and upper end of energy sensitivity [keV] assuming an E^-2 spectrum (E^2 dN/dE)"
}

For unification, shouldn't it be in keV units?

Reply:
I had this initially as using one of the core classes for energy range, but it was removed recently.  I can use keV, for all values, but GeV/TeV are much more appropriate for the neutrinos we're sensitive to.  Please advise if we should adapt all units

Reply:
I agree about making this an array, but we've discussed it, and use whatever units you'd like, as long as they agree with the description.

Reply:
Converted to an array, units to GeV to match flux units.